TestAssets/TestPackages/dotnet-dependency-context-test/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Linq;
+using Microsoft.Extensions.DependencyModel;
+
+namespace Microsoft.DotNet.Tools.DependencyInvoker
+{
+    public class Program
```

Comment:
But seriously, it was tricky enough for me to work out one kind of tests. I'd just like to get this in :)